### PR TITLE
WIP: init Text Component

### PR DESCRIPTION
**Pull request by smankow**

### CODE REVIEW

File: src/asset/icon/ErrorIcon.jsx [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+      <path
+        strokeLinecap="round"
+        strokeLinejoin="round"
+        d="M12 8v4m0 4h.01M21 12a9 9 0 11-18 0 9 9 0 0118 0z"
+      />
+    </svg>
+  );
+};
+
+export default ErrorIcon;
```
Comment (Patrick-FRE) [inline src/asset/icon/ErrorIcon.jsx:24]: Should not put JSX inside Assets folder.

File: src/components/Text/OldText.css [hunk 2]
```diff
@@ -0,0 +1,66 @@
+/* * {
```
Comment (Patrick-FRE) [inline src/components/Text/OldText.css:1]: Remove it if you are not using it.

File: src/components/Text/Text.tsx [hunk 3]
```diff
@@ -0,0 +1,40 @@
+import React, { InputHTMLAttributes} from 'react';
+import ErrorIcon from "../../asset/icon/ErrorIcon";
+// import errorIcon from "../../asset/icon/error-icon.svg"
```
Comment (Patrick-FRE) [inline src/components/Text/Text.tsx:3]: remove the unused comments

File: src/components/Text/_Text.scss [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+.textbox.isError{
+    // @include textbox-size(
+    //     1rem,
+    //     1rem,
+    //     1rem
+    // );
+    @include textbox-style(
+        $white-pink,
+        $gray,
+        $black
```
Comment (Patrick-FRE) [inline src/components/Text/_Text.scss:29]: Wrong color

File: src/components/Text/Text.tsx [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+  return (
+    <>
+      <div className="textbox-container">
+        <input
+          className={"textbox " + inputClass}
+          value={inputValue}
+          onChange={(e) => setInputValue(e.target.value)}
+          {...rest}
+        />
+        {error && <ErrorIcon className="svg-error" />}
```
Comment (Patrick-FRE) [inline src/components/Text/Text.tsx:31]: you can just use <svg> inside Text Component

File: src/components/Text/Text.tsx [hunk 6]
```diff
@@ -1,37 +1,62 @@
...
+  if (className) {
+    styleClasses += ' ' + className;
   }
 
   const [inputValue, setInputValue] = useState(value);
+  useEffect(() => setInputValue(value), [value])
+  const changeValue = (e: React.ChangeEvent<HTMLInputElement>) => {
+    setInputValue(e.currentTarget.value);
+
+    if (onChange) {
```
Comment (ylwfleece) [inline src/components/Text/Text.tsx:31]: may be redundant having onChange here, should update ui regardless

**Pull request closed without merging**